src/test/java/com/wikia/webdriver/common/contentpatterns/WikiTextContent.java

```diff
@@ -35,6 +35,8 @@
   public static final String EXT_LINK_PREFIX = "[";
   public static final String EXT_LINK_POSTFIX = "]";
 
+  public static final String REDIRECT = "#redirect";
```

Comment:
this is standard wikitext redirect markup, so the context is given by the class which contains this field: WikiTextContent.
This is just a wikitext redirect: https://www.mediawiki.org/wiki/Help:Redirects#Creating_a_redirect